packages/chakra-ui/src/AltDateWidget/AltDateWidget.tsx

```diff
@@ -11,12 +11,19 @@ import {
 } from '@rjsf/utils';
 import { getChakra } from '../utils';
 
-function AltDateWidget<T = any, S extends StrictRJSFSchema = RJSFSchema, F extends FormContextType = any>(
-  props: WidgetProps<T, S, F>,
-) {
-  const { autofocus, disabled, id, onBlur, onFocus, options, readonly, registry } = props;
+function AltDateWidget<T = any, S extends StrictRJSFSchema = RJSFSchema, F extends FormContextType = any>({
+  autofocus = false,
+  disabled = false,
+  readonly = false,
+  time = false,
+  options = {
+    yearsRange: [1900, new Date().getFullYear() + 2],
```

Comment:
Defaults can behave differently around `options` props than destructure initialization, due to the code that is in the `getWidget()` implementation in `@rjsf/utils`. You may want to deal with that too... See `mergeWidgetOptions()` in `getWidget.ts`